source/gent/tier1_hortense.rst

```diff
@@ -383,6 +460,45 @@ and load one or more modules via the ``module load`` command to start using them
 If software that you require is missing, please submit a software installation request
 via https://www.ugent.be/hpc/en/support/software-installation-request .
 
+.. _hortense_software_readonly:
+
+Accessing software via ``/readonly`` mount point
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+The central software stack on Tier-1 Hortense is provided via the ``/readonly`` mount point
+(see also :ref:`hortense_accessing_data_readonly`). This is largely transparent as long as you
+only load modules that are part of the central software stack.
+
+If you install any software yourself in your project scratch directory, we highly recommend
+you to also access it only through the ``/readonly`` mount point, since this can have a significant
+performance benefit.
+
+To ensure that the paths which are 'engraved' in your own software installations always start with ``/readonly/``,
+for example in scripts or binaries that make part of the installation,
+you should install the software using the ``bwrap`` utility. This allows you to "rename" the path to your
+project scratch directory so it starts with ``/readonly/``, while preserving write access to it.
+
+Assuming that the procedure to install the software is implemented in a script named ``install.sh``,
+you can use ``bwrap`` as follows:
+
+.. code::
+
+   bwrap --bind / / --bind $VSC_SCRATCH_PROJECTS_BASE /readonly/$VSC_SCRATCH_PROJECTS_BASE --dev /dev --bind /dev/log /dev/log ./install.sh
+
+The ``install.sh`` script should be implemented such that it installs the software to
```

Comment:
you can also suggest to start a shell with bwrap, and then install the software from the shell

Reply:
I'd say that's harder to explain... Also, we don't want to encourage people to install stuff manually.

Reply:
fixed in 9ceb1e7